Bellerophon/Bellerophon/Sources/BPManager.swift

```diff
@@ -67,60 +89,48 @@ public class BellerophonManager: NSObject {
                 // An error occurred
                 completionHandler(result: .Failed)
             }
-        })
+        }
     }
 
     // MARK: Private Methods
-    override init() {
-        super.init()
-        NSNotificationCenter.defaultCenter().addObserver(self, selector: Selector(stopTimer()), name: UIApplicationDidEnterBackgroundNotification, object: nil)
-    }
-
-    deinit {
-        NSNotificationCenter.defaultCenter().removeObserver(self)
-    }
-
-    func handleAppStatus(status: BellerophonStatusProtocol) {
+    private func handleAppStatus(status: BellerophonStatusProtocol) {
         if status.apiInactive() {
-            self.displayKillSwitch(status.userMessage())
```

Comment:
The user message is not displayed by `displayKillSwitch()`. It has to be displayed by the app itself.